src/App.js

```diff
@@ -1,25 +1,182 @@
+
+  useEffect( () => {
+    const database = getDatabase(firebase)
+    const dbRef = ref(database)
+    onValue(dbRef, (response) => {
+      const data = response.val()
+      const {userScore, computerScore} = data
+      setUserValue(userScore)
+      setComputerValue(computerScore)
+    })
+  },[])
+
+  const increaseUserScore = () => {
+    const database = getDatabase(firebase)
+    const dbRef = ref(database)
+    get(dbRef).then((data) => { 
+      const oldData = data.val()
+      const newData = {
+        userScore: oldData.userScore +1,
+        drawScore: oldData.drawScore,
+        computerScore: oldData.computerScore
+      }
+      update(dbRef, newData)
+    }).catch((error) => {
+      console.log(error);
+    })
+  
+  }
+
+    const increaseComputerScore = () => {
+    const database = getDatabase(firebase)
+    const dbRef = ref(database)
+    get(dbRef).then((data) => { 
+      const oldData = data.val()
+      const newData = {
+        computerScore: oldData.computerScore +1,
+        drawScore: oldData.drawScore,
+        userScore: oldData.userScore
+      }
+      update(dbRef, newData)
+    }).catch((error) => {
+      console.log(error);
+    })
+  
+  }  
+  
+    const increaseDrawScore = () => {
+    const database = getDatabase(firebase)
+    const dbRef = ref(database)
+    get(dbRef).then((data) => { 
+      const oldData = data.val()
+      const newData = {
+        drawScore: oldData.drawScore +1,
+        userScore: oldData.userScore,
+        computerScore: oldData.computerScore
+      }
+      update(dbRef, newData)
+    }).catch((error) => {
+      console.log(error);
+    })
+  
+  }
+
+  const handleClick = (value) =>  {
+    setUserChoice(value)
+    generateComputerChoice()
+  } 
+
+  useEffect(() => {
+  
+    switch(userChoice + computerChoice) {
+      default:
+        setResult('Choose your weapon')
+        break
+
+      case 'scissorspaper':
+        setResult('YOU WIN!')
+        increaseUserScore()
+        break
+
+      case 'rockscissors':
+        setResult('YOU WIN!')
+        increaseUserScore()
+        break
+
+      case 'paperrock':
+        setResult('YOU WIN!')
+        increaseUserScore()
+        break
+
+      case 'paperscissors':
+        setResult('YOU LOSE!')
+        increaseComputerScore()
+        break
+
+      case 'scissorsrock':
+        setResult('YOU LOSE!')
+        increaseComputerScore()
+        break
+
+      case 'rockpaper':
+        setResult('YOU LOSE!')
+        increaseComputerScore()
+        break
+
+      case 'rockrock':
+        setResult('IT\'S A DRAW!')
+        increaseDrawScore()
+        break
+
+      case 'paperpaper':
+        setResult('IT\'S A DRAW!')
+        increaseDrawScore()
+        break
+
+      case 'scissorsscissors':
+        setResult('IT\'S A DRAW!')
+        increaseDrawScore()
+        break
+    }
+  
+  }, [computerChoice, userChoice])
+
+  const generateComputerChoice = () => {
```

Comment:
just for code readability you might want to store the computer choice function above with your other function definitions